tests/python/relax/test_tvmscript_printer_relax.py

```diff
@@ -73,6 +94,40 @@ def func(a: R.Tensor((10, 10))) -> R.Tensor((10, 10)):
     )
 
 
+def test_nested_function():
```

Comment:
This actually uncovered a seemingly unrelated bug in the printer, where the nested function would be printed with a suffix after its name (see the changes in `src/script/printer/utils.h`) because the LHS variable in the `VarBinding` was treated as already having been encountered. Also fixed.